assignments/prototypes.js

```diff
@@ -32,6 +55,20 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(humanoidAttributes) {
+  this.team = humanoidAttributes.team;
+  this.weapons = humanoidAttributes.weapons;
+  this.language = humanoidAttributes.language;
+  CharacterStats.call(this,humanoidAttributes);
```

Comment:
its probably safer to put your call before setting your properties, 
if its below it can cause bugs sometimes.